classes/patterns/class-issues.php

```diff
@@ -28,16 +28,18 @@ public static function get_name(): string {
 	 */
 	public static function get_media_text_template(): string {
 		$media_link = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-40x40.jpg';
-		return '
-			<!-- wp:media-text {"mediaWidth":15,"mediaLink":"' . $media_link . '","mediaType":"image","mediaSizeSlug":"thumbnail","isStackedOnMobile":false,"verticalAlignment":"center","imageFill":false,"backgroundColor":"white","className":"is-style-large-padding force-no-lightbox"} -->
-			<div class="wp-block-media-text alignwide is-vertically-aligned-center has-white-background-color is-style-large-padding has-background force-no-lightbox" style="grid-template-columns:15% auto">
+
+		return '<!-- wp:group {"style":{"spacing":{"padding":{"top":"16px","right":"16px","bottom":"16px","left":"16px"}}},"backgroundColor":"white"} -->
+			<div class="wp-block-group has-white-background-color has-background" style="padding-top:16px;padding-right:16px;padding-bottom:16px;padding-left:16px">
+			<!-- wp:media-text {"mediaLink":"' . $media_link . '","mediaType":"image","mediaWidth":15,"mediaSizeSlug":"thumbnail","isStackedOnMobile":false,"verticalAlignment":"center","imageFill":false,"className":"force-no-lightbox is-style-default"} -->
```

Comment:
You're right! In fact, we should remove this registered class too. Because we have [this](https://github.com/greenpeace/planet4-master-theme/pull/1765/files#diff-160c735a986cf8c05e1f6b2e9ac89dd0410562d2fe5ccbdc93157bd92f3dc2bcR91-R95).